forge/db/models/Application.js

```diff
@@ -70,7 +70,17 @@ module.exports = {
                                     required: false
                                 }
                             ]
-                        })
+                        }
+
+                        if (includeInstanceStorageFlow) {
+                            // Used for instance status
+                            include.include.push({
+                                model: M.StorageFlow,
+                                attributes: ['id', 'flow', 'updatedAt']
```

Comment:
At first glance, I can't see anywhere we use `flow` from this - given that *could* be sizeable it would be waste of effort to retrieve if all we want is the `updatedAt` column

Reply:
Agreed, `flow` shouldn't be there!

Reply:
[4b93619](https://github.com/FlowFuse/flowfuse/pull/2955/commits/4b93619b91b2a80ad94a8c50d6225a4cffb7284a)